source/includes/stackpath/_workloads.md

```diff
@@ -0,0 +1,115 @@
+## Workloads
+
+Deploy and manage your workloads.
+
+<!-------------------- LIST WORKLOADS -------------------->
+
+### List workloads
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/stackpath/test-area/workloads"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "id": "1b932678-1038-4ab4-9fa4-c4c06e696e20",
+      "name": "stackpath_wl_01",
+      "stackId": "3e651d2e-b1f4-44ad-b21a-ebcd0bf79ca1",
+      "slug": "stackpath-workload-01",
+      "status": "ACTIVE",
+      "spec": "SP-2",
+      "version": "3",
+      "created": "2020-11-06T18:48:36.850115786Z",
+      "type": "VM",
+      "network": "default",
+      "cpu": "2",
+      "memory": "4Gi",
+      "isRemoteManagementEnabled": false,
+      "image": "stackpath-edge/centos-7:v202007311835"
+    }
+  ],
+  "metadata": {
+    "recordCount": 1
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/workloads</code>
+
+Retrieve a list of all workloads in a given [environment](#administration-environments).
+
+Attributes | &nbsp;
+------- | -----------
+`id`<br/>*string* | A workload's unique identifier.
+`name`<br/>*string* | The display name of the workload.
+`stackId`<br/>*string* | The ID of the stack that a workload belongs to.
+`slug`<br/>*string* | A workload's programmatic name. Workload slugs are used to build its instances names.
+`status`<br/>*string* | The status of the workload. It can be either ACTIVE or DISABLED.
+`spec`<br/>*string* | Specification type for resources which are allocated to each instance in a workload.
+`version`<br/>*string* | A version number for the workload. Versions start at 1 when they are created and increment by 1 every time they are updated.
+`created`<br/>*string* | Creation timestamp of the workload.
+`type`<br/>*string* | Specify whether a workload is a VM based workload or container based.
```

Comment:
I think it is VM-based and container-based. We would need to double check

Reply:
If yes, then the same changes needs to be applied below

Reply:
A workload can be either of VM type or of container type.
![image](https://user-images.githubusercontent.com/5191515/98833180-ba134f80-240b-11eb-89cd-2a0f59f0e110.png)

Reply:
I meant "VM-based" instead of "VM based". Sorry, it was ambiguous

Reply:
ooh...My bad. I misunderstood.
I am making those changes.

Reply:
f7751db